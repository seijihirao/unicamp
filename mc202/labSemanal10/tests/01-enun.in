i 1703980530 13
i 1 31
i 285942093 51
i 1703980530 29
i 967869928 31
b 1291764354
b 1703980530
b 967869928
b 1
r 967869928
b 967869928
r 1703980530
b 1703980530
f
