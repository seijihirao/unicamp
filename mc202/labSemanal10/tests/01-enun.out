1291764354 nao existe
valor para 1703980530: 29
valor para 967869928: 31
valor para 1: 31
967869928 nao existe
1703980530 nao existe
